doc/server/aws.md

```diff
@@ -66,6 +66,19 @@ output the url of the created API Gateway which you can call followed by `/api/h
 
 To destroy all the created resources run `terraform destroy`.
 
+### CDK
```

Comment:
We have two distinct deployment options for tapir-serverless on AWS: SAM and CDK, so I think we'd need some guide at the top as to what are the responsibilities of the various modules, and in which combinations they should be used, and which dependencies to add. Right now I think it would be challenging to a new user to understand how a serverless tapir endpoint can be deployed.

Reply:
As a side note, Examples > SAM and Examples > SAM Example are confusing ;) (not directly connected to CDK, but might be good to cleanup up as well)

Reply:
I changed documentation, feel free to check whether it makes sense :)